lib/YaST/NetworkSettings/NetworkCardSetup/DeviceTypeDialog.pm

```diff
@@ -26,8 +26,6 @@ sub select_device_type {
         bond => 'alt-o',
         vlan => 'alt-v'
     };
-    record_soft_failure('bsc#1191112 - Resizing window as workaround for YaST content not loading');
-    send_key_until_needlematch(DEVICE_TYPE_DIALOG, 'alt-f10', 10, 2);
```

Comment:
missing `assert_screen(DEVICE_TYPE_DIALOG);`
we need to check always the previous code fore all cases or at least is clear that needle still needs to asserted after removing the workaround.